aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -709,6 +710,27 @@ impl AptosVM {
 
         let func = session.load_script(script.code(), script.ty_args())?;
 
+        // Check that unstable bytecode cannot be executed on mainnet
+        if self
+            .features()
+            .is_enabled(FeatureFlag::REJECT_UNSTABLE_BYTECODE_FOR_SCRIPT)
+        {
+            let script = match CompiledScript::deserialize_with_config(
```

Comment:
Can we cache the deserialization of compiled scripts here by bundling it with `verify_no_event_emission_in_script` (this also deserializes the script) under a single function which does the deserialization and then runs verification passes?

Reply:
done

Reply:
Why else branch in the code below? I think having
```
// deserialize here
if feature_enabled {
  self.reject_unstable_bytecode_for_script(&script)?;
}
verifier::event_validation::verify_no_event_emission_in_compiled_script(&script)?;
```
is sufficient?

Reply:
done